manifests/init.pp

```diff
@@ -504,7 +504,7 @@
       hasrestart => $trap_service_hasrestart,
       require    => [ Package['snmpd'], File['var-net-snmp'], ],
     }
-  } elsif $::osfamily == 'Suse' {
+  } elsif $::operatingsystem =~ /(?i:opensuse|suse)/ {
```

Comment:
Is this supposed to be `$::operatingsystem` or `$::osfamily`?  manifests/client.pp is using `$::osfamily`.